source/2019-11-01-the-ember-times-issue-122.md

```diff
@@ -87,13 +88,13 @@ Curious to learn more or even contribute to build processes for Ember learning m
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Ember Simple Auth 2.0.0 release 🛳](https://twitter.com/simplabs/status/1189264438026747907)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+This week the good people over at [@simplabs](https://github.com/simplabs) released [version 2.0.0](https://github.com/simplabs/ember-simple-auth/releases/tag/2.0.0) of the excellent authentication/authorization library **Ember Simple Auth**.
```

Comment:
I added a comma below. After this sentence, can we acknowledge the six people who contributed to the 2.0 release?
```suggestion
This week, the good people over at [@simplabs](https://github.com/simplabs) released [version 2.0.0](https://github.com/simplabs/ember-simple-auth/releases/tag/2.0.0) of the excellent authentication/authorization library **Ember Simple Auth**.
```

Reply:
Good call

Reply:
@ijlee2 Are you thinking we call them individually?

Reply:
Yeah. Maybe by their GitHub handles only since we have 6 people? You can find the handles in https://github.com/simplabs/ember-simple-auth/releases/tag/2.0.0